concordium-contracts-common/src/schema_json.rs

```diff
@@ -964,6 +999,109 @@ impl Type {
     ) -> Result<(), JsonError> {
         write_bytes_from_json_schema_type(schema, json, out)
     }
+
+    pub fn schema_type_to_json(&self) -> serde_json::Value {
```

Comment:
It would be good with a few tests, at least for some of the more complex types.
I am not sure what would be best, asserting on the strings, or just on the JSON types.

Reply:
I don't think it's specified the order in which JSON fields are output in objects, so those tests will be fragile if we compare the output strings. We should compare `Value`s.